07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,42 @@
+*{
+  font-family:sans-serif;
+}
+
+.my_image{
+  background-image: url("../css/my-style.css");
```

Comment:
the background image needs to get into the img folder. so instead of ../css try locating the image through ../img